javascript/queueWithStacks/README.md

```diff
@@ -0,0 +1,15 @@
+# Code Challenge: Linked List Zip
+
+ > Create a Pseudo Queue class with two Stack objects as nodes. Create enqueue and dequeue methods.
+
+## CC=Code Challenge/Pair Programming
+
+> Tina Myers, Stacy Burris & Sara Strasner
+
+## Languages:
+
+javaScript
+
+## Solution:
+
+[whiteboard image](../assets/queueWithStack.png)
```

Comment:
You are off to a great start Tina! I would like to see some code added and tests as well!